website/docs/guides/sl-cookbook-recipes.md

```diff
@@ -0,0 +1,648 @@
+   ```
+
+<!-- Cookbook 3 -->
+
+## How to calculate CAC using metrics
+
+Use this cookbook recipe to understand how to [calculate customer acquisition cost (CAC)](https://en.wikipedia.org/wiki/Customer_acquisition_cost) using ratio metrics in dbt.
+
+- [Objectives](#objectives-2)
+- [Use cases](#use-case-2)
+- [Step 1: Define semantic models](#step-1-define-semantic-models-2)
+- [Step 2: Define the CAC metric](#step-2-define-the-cac-metric)
+- [Step 3: Validate and query your metrics](#step-3-validate-and-query-metric-1)
+
+### Objectives
+
+- Understand how to configure and use semantic models in dbt to track and report customer acquisition costs.
+- Learn how to define and calculate CAC by dividing the total acquisition cost by the number of new customers.
+
+### Use case
+Track and analyze the cost of acquiring new customers to optimize marketing and sales strategies.
+
+:::tip
+💡 Note that we’re using the [parallel subfolder approach](/best-practices/how-we-build-our-metrics/semantic-layer-7-semantic-structure#files-and-folders) for better organization in large projects, which means you’re creating a semantic model folder containing all your semantic layer code. 
+
+Alternatively, you can use the [one-YAML-per-marts-model approach](/best-practices/how-we-build-our-metrics/semantic-layer-7-semantic-structure#files-and-folders) which puts documentation, data tests, unit tests, semantic models, and metrics into a unified file corresponding to a dbt-modeled mart.
+:::
+
+### Step 1: Define semantic models
+Create a YAML file under the `semantic_models` folder: (`models/semantic_models/sem_customer_acquisition.yml`) to define a semantic model for customer acquisition data.
+
+<File name="models/semantic_models/sem_customer_acquisition.yml">
+
+```yaml
+semantic_models:
+  - name: sem_customer_acquisition
+    description: |
+      This model aggregates customer acquisition data, providing metrics for business to analyze the cost of acquiring new customers.
+    model: ref('fct_customer_acquisition')
+    defaults:
+      agg_time_dimension: date_month
+    entities:
+      - name: date
+        type: primary
+        expr: date_month
+    measures:
+      - name: total_acquisition_cost
+        agg: sum
+        expr: acquisition_cost
+        agg_time_dimension: date_month
+      - name: new_customers
+        agg: sum
+        expr: customer_count
+        agg_time_dimension: date_month
+    dimensions:
+      - name: date_month
+        type: time
+        type_params:
+          time_granularity: month
+      - name: acquisition_channel
+        type: categorical
+```
+
+</File>
+
+### Step 2: Define the CAC metric
+Create another YAML file under the metrics folder: (`models/metrics/sem_customer_acquisition_metrics.yml`) to define the CAC metric.
+
+<File name="models/metrics/sem_customer_acquisition_metrics.yml">
+
+```yaml
+metrics:
+  - name: cac
+    description: "Customer acquisition cost"
+    label: 'CAC'        
+    type: ratio
```

Comment:
`ratio` metrics require metrics as inputs for numerator, denominator. So we will need to define those metrics too (right now it seems like we are trying to use the measures)